distributions/startupway/src/server/script.ts

```diff
@@ -27,29 +36,29 @@ export async function dbcreate() {
 				rowsAsArray:true
 			});
 			conn = await pool.getConnection();
-			//users
+			// users
 			await conn.query(`CREATE TABLE \`users\` (\`userId\` int(11) NOT NULL AUTO_INCREMENT,\`firstName\` varchar(50) NOT NULL,\`lastName\` varchar(50) NOT NULL,\`username\` varchar(120) NOT NULL,\`password\` varchar(90) NOT NULL,\`email\` varchar(100) NOT NULL,\`phone\` varchar(15) NOT NULL,\`birthDate\` datetime NOT NULL,\`avatarUu\` varchar(100) NOT NULL,\`socialMedia\` text NOT NULL,\`userDetails\` text NOT NULL,\`role\` text NOT NULL,\`lastLogin\` datetime NOT NULL,PRIMARY KEY (\`userId\`),UNIQUE KEY \`username\` (\`username\`),UNIQUE KEY \`username_password\` (\`username\`,\`password\`)) ENGINE=InnoDB AUTO_INCREMENT=78`)

```

Comment:
Foloseste " si `, sa nu mai tot punem `

```suggestion
			await conn.query("CREATE TABLE `users` (`userId` int(11) NOT NULL AUTO_INCREMENT,\`firstName\` varchar(50) NOT NULL,\`lastName\` varchar(50) NOT NULL,\`username\` varchar(120) NOT NULL,\`password\` varchar(90) NOT NULL,\`email\` varchar(100) NOT NULL,\`phone\` varchar(15) NOT NULL,\`birthDate\` datetime NOT NULL,\`avatarUu\` varchar(100) NOT NULL,\`socialMedia\` text NOT NULL,\`userDetails\` text NOT NULL,\`role\` text NOT NULL,\`lastLogin\` datetime NOT NULL,PRIMARY KEY (\`userId\`),UNIQUE KEY \`username\` (\`username\`),UNIQUE KEY \`username_password\` (\`username\`,\`password\`)) ENGINE=InnoDB AUTO_INCREMENT=78")

```